.github/workflows/ci.yaml

```diff
@@ -0,0 +1,147 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+name: Gridiron
+
+jobs:
+  check:
+    name: Check
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - beta
+          - 1.38.0 #MSRV
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: ${{ matrix.rust }}
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          command: check
+
+  build_linux:
+    name: (Linux) Platform Builds 
+    runs-on: ubuntu-18.04
+    strategy:
+      matrix:
+        target:
+          - armv7-linux-androideabi     # Android ARM7
+          - aarch64-linux-android       # Android x64
+          - x86_64-unknown-linux-musl   # Alpine Linux x86_64
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          target: ${{ matrix.target }}
+          override: true
+      - uses: actions-rs/cargo@v1
+        with:
+          use-cross: true
+          command: build
+          args: --release --target=${{ matrix.target }}
+
+  build_macos:
+    name: (MacOS X) Platform Builds 
+    runs-on: macos-10.15
+    strategy:
+      matrix:
+        target:
+          - armv7-apple-ios
+          - aarch64-apple-ios
```

Comment:
These are the two mobile architectures, right? Do you also want to build on x86_64 for Mac? That's what the name of this job implies.

Reply:
Good catch. I was able to add `x86_64-apple-darwin` to cross.